src/service_client/assisted_service_api.py

```diff
@@ -328,7 +328,7 @@ def download_cluster_events(self, cluster_id: str, output_file: str, categories=
         with open(output_file, "wb") as _file:
             events = self.get_events(cluster_id, categories=categories)
             _file.write(json.dumps(events, indent=4).encode())
-            self._events_junit_exporter.collect(events, suite_name="cluster_events", xml_suffix=cluster_id)
```

Comment:
The `xml_suffix` is just setting the xml file name and not the test suite/case name

Reply:
When we have multiple clusters on single test we don't want to override the report

Reply:
To fix that issue we might need to change the format https://github.com/openshift/assisted-test-infra/blob/d005d3271ad2aa38ae3ea71f08a82fd8deec0886/src/service_client/assisted_service_api.py#L39